packages/esbuild/lib/index.js

```diff
@@ -142,6 +142,7 @@ export function esbuild(options) {
       return {
         contents: value || '',
         errors,
+        loader: options?.jsx ? 'jsx' : 'js',
```

Comment:
I think it’s good to put a `const settings = {...options, SourceMapGenerator}` on L69, *use* that `settings` in what is now on L69, and then also use `settings.jsx` here. Instead of this optional chaining

Reply:
Done!